libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -1115,11 +1073,13 @@ void AC_PosControl::run_xy_controller(float dt)
 void AC_PosControl::accel_to_lean_angles(float accel_x_cmss, float accel_y_cmss, float& roll_target, float& pitch_target) const
 {
     float accel_right, accel_forward;
+    float sin_yaw = sinf(_attitude_control.get_att_target_euler().z);
+    float cos_yaw = cosf(_attitude_control.get_att_target_euler().z);
 
     // rotate accelerations into body forward-right frame
     // todo: this should probably be based on the desired heading not the current heading
-    accel_forward = accel_x_cmss * _ahrs.cos_yaw() + accel_y_cmss * _ahrs.sin_yaw();
-    accel_right = -accel_x_cmss * _ahrs.sin_yaw() + accel_y_cmss * _ahrs.cos_yaw();
+    accel_forward = accel_x_cmss * cos_yaw + accel_y_cmss * sin_yaw;
```

Comment:
```suggestion
    float accel_forward = accel_x_cmss * cos_yaw + accel_y_cmss * sin_yaw;
```

and remove the lines above - same for accel_right

Reply:
changed to use const float, txs